docs/entrypoints.md

```diff
@@ -0,0 +1,56 @@
+# Entrypoints
+
+This document describes the various ways that weave as a binary gets launched, and where and how it processes options.
+
+The goal is to enable maintainers to modify functionality without having to reverse engineer the code to find all of the places you can start a weave binary and what it depends upon.
+
+## Components
+### weaveutil
+weaveutil is the primary binary for managing a existing weave network. It gets information about the network, attaches and detaches containers, etc.
```

Comment:
Suggest "`weaveutil` provides a number of functions for managing a existing weave network"